internal/pkg/consensus/expected.go

```diff
@@ -216,7 +216,7 @@ func (c *Expected) validateMining(ctx context.Context,
 		}
 
 		// Verify EPoSt VRF proof ("PoSt randomness")
-		if err := c.VerifyEPoStVrfProof(ctx, blk.Parents, blk.Height, blk.Miner, workerSignerAddr, blk.EPoStInfo.VRFProof); err != nil {
+		if err := c.VerifyEPoStVrfProof(ctx, blk.Parents, blk.Height-1, blk.Miner, workerSignerAddr, blk.EPoStInfo.VRFProof); err != nil {
```

Comment:
This -1 could use some explanation in a comment or assign it to a variable that names it  (`parentHeight`?).

Reply:
yeah I should really use the election post lookback constant here